@xen-orchestra/web-core/lib/components/ui/toaster/UiToaster.vue

```diff
@@ -0,0 +1,103 @@
+<!-- v3 -->
+<template>
+  <div :class="toVariants({ accent })" class="ui-toaster">
+    <div class="label">
```

Comment:
After re-reviewing the component, I don't think the class name is the most appropriate. Maybe something like `first-line` would provide more information about what this element represents?

Reply:
Why not something like :
`<template>
  <div :class="toVariants({ accent })" class="ui-toaster">
    <div class="toast-content">
      <VtsIcon class="information-icon" :accent :icon="faCircle" :overlay-icon="icon" />
      
      <div class="content">
        <div class="typo h5-semi-bold">
          <slot />
        </div>
        <div v-if="slots.description" class="description typo p2-regular">
          <slot name="description" />
        </div>
      </div>
      
      <UiButtonIcon class="close-icon" :icon="faXmark" :accent size="medium" />
    </div>
    
    <div v-if="slots.actions" class="toast-actions">
      <slot name="actions" />
    </div>
  </div>
</template>`